jni/lib/src/jarray.dart

```diff
@@ -92,10 +91,9 @@ class JArray<E> extends JObject {
   /// Creates a [JArray] of the given length with [fill] at each position.
   ///
   /// The [length] must be a non-negative integer.
-  /// The [fill] must be a non-null [JObject].
   static JArray<E> filled<E extends JObject>(int length, E fill) {
-    assert(!fill.isNull, "fill must not be null.");
-    final clazz = fill.getClass();
+    RangeError.checkNotNegative(length);
+    final clazz = fill.$type.getClass();
```

Comment:
ArgumentError on `fill.isNull`?

Reply:
`fill` now can be null.

Reply:
Right, we use the Dart runtime type now.